src/polygonfill2d.jl

```diff
@@ -22,6 +22,26 @@ function (f::BoundaryFill)(
     res
 end
 
+"""
+    (f::FloodFill)(res::AbstractArray{T,2}, verts::Vector{CartesianIndex{2}}, x::Int, y::Int, fill_value::T, boundary_value::T) where {T <: Colorant}
+
+"""
+
+function (f::FloodFill)(
+    res::AbstractArray{T,2},
+    verts::Vector{CartesianIndex{2}},
+    x::Int,
+    y::Int,
+    ) where {T<:Colorant}
+    if (res[y, x] != f.current_value || res[y, x] == f.fill_value) return end
```

Comment:
Can we get `current_value` through something like `res[x, y]` (or `res[y,x]`)?

Reply:
Yes, that can be done. We want to set the value of current_value at time when struct creation is called, but at that time res is not accessible  to struct . There will be quick way to solve it 😄 , I need to lookup .